### PR TITLE
Adding swot_simulator

### PR DESCRIPTION
**Pull request by fbriol**

### CODE REVIEW

File: recipes/swot_simulator/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  version: {{ version }}
+
+source:
+    url: https://github.com/CNES/swot_simulator/releases/download/{{ version }}/swot_simulator-{{ version }}.tar.gz
+    sha256: 6e5ad18694cc774c1b2affc7f01bc1073e5392ecaa4ce50049f9c0bce5b6f85b
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<36]
```
Comment (hmaarrfk) [inline recipes/swot_simulator/meta.yaml:15]:
```suggestion
  noarch: python
```

File: recipes/swot_simulator/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+requirements:
+  host:
+    - pip
+    - python
```
Comment (hmaarrfk) [inline recipes/swot_simulator/meta.yaml:20]:
```suggestion
    - python >=3.6
```

File: recipes/swot_simulator/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    - setuptools
+  run:
+    - dask
+    - distributed
+    - mkl_fft
+    - netcdf4
+    - numba
+    - {{ pin_compatible('numpy') }}
+    - pyinterp
+    - python {{ python }}
```
Comment (hmaarrfk) [inline recipes/swot_simulator/meta.yaml:30]:
```suggestion
    - python >=3.6
```

File: recipes/swot_simulator/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    - pip
+    - python
+    - setuptools
+  run:
+    - dask
+    - distributed
+    - mkl_fft
+    - netcdf4
+    - numba
+    - {{ pin_compatible('numpy') }}
```
Comment (hmaarrfk) [inline recipes/swot_simulator/meta.yaml:28]:
```suggestion
    - numpy
```

**Comment by fbriol** [comment]
Now the generation doesn't work on Windows anymore. The addition of this ``noarch: python`` option seems to be the cause of the problem. The error message in the log is: ``[noarch_python] Noarch package contains binary script: swot_simulator.exe``.

**Comment by hmaarrfk** [comment]
Sorry, I didn't notice the entry points. You need to specify them in your build section.

**Comment by hmaarrfk** [comment]
https://github.com/conda-forge/staged-recipes/pull/11691/files#diff-14d7bf315a71f830388faf62d6b505deR17

**Comment by hmaarrfk** [comment]
cool stuff. Somebody else will have to come in and merge

**Pull request merged**